### PR TITLE
[Databricks] Support CMK

### PR DESCRIPTION
**Pull request by isra-fel**
## Description

Added support for CMK (Customer managed keys)

## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by adxsdkps** [comment]
Can one of the admins verify this patch?

File: src/Databricks/Az.Databricks.psd1 [hunk 1]
```diff
@@ -1,133 +1,24 @@
-#
-# Module manifest for module 'Az.Databricks'
-#
-# Generated by: Microsoft Corporation
-#
-# Generated on: 3/24/2020
-#
-
 @{
```
Comment (dingmeng-xue) [inline src/Databricks/Az.Databricks.psd1:1]: this file needs to be changed manually. Given no new cmdlet, it needs to be rollback

File: src/Databricks/Az.Databricks.csproj [hunk 2]
```diff
@@ -1,7 +0,0 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
```
Comment (dingmeng-xue) [inline src/Databricks/Az.Databricks.csproj:1]: This file should not be changed.

File: src/Databricks/Az.Databricks.psm1 [hunk 3]
```diff
@@ -24,17 +24,17 @@
       }
     }
     if(-not $accountsModule) {
-      $hasAdequateVersion = (Get-Module -Name $accountsName -ListAvailable | Where-Object { $_.Version -ge [System.Version]'1.7.2' } | Measure-Object).Count -gt 0
+      $hasAdequateVersion = (Get-Module -Name $accountsName -ListAvailable | Where-Object { $_.Version -ge [System.Version]'1.7.4' } | Measure-Object).Count -gt 0
```
Comment (dingmeng-xue) [inline src/Databricks/Az.Databricks.psm1:27]: https://github.com/Azure/autorest.powershell/issues/609

Comment (dingmeng-xue) [reply src/Databricks/Az.Databricks.psm1:27]: we don't need to fix it in this PR. I'm OK for the time being.

File: src/Databricks/ChangeLog.md [hunk 4]
```diff
@@ -1,24 +0,0 @@
-<!--
-    Please leave this section at the top of the change log.
```
Comment (dingmeng-xue) [inline src/Databricks/ChangeLog.md:2]: Not generated. follow standard way.

File: src/Databricks/Databricks.sln [hunk 5]
```diff
@@ -1,55 +0,0 @@
-Microsoft Visual Studio Solution File, Format Version 12.00
```
Comment (dingmeng-xue) [inline src/Databricks/Databricks.sln:1]: keep original version.

File: src/Databricks/Properties/AssemblyInfo.cs [hunk 6]
```diff
@@ -1,28 +0,0 @@
-// ----------------------------------------------------------------------------------
```
Comment (dingmeng-xue) [inline src/Databricks/Properties/AssemblyInfo.cs:1]: keep original version.

**Pull request merged**